shopinvader_anonymous_partner/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+- some cron to remove old anonymous partners
```

Comment:
IMO we should also provide a way to filter out by default anonymous partners into the contact's views. And a pre defined filter to list all anonymous partners....

Reply:
anonymous partner should be inactive by default, we only active them when we fill real information.

Reply:
> anonymous partner should be inactive by default, we only active them when we fill real information.

@sebastienbeau ok, I'll update this PR to do that.

Regarding activation, I'm now thinking the normal flow should not activate an anonymous partner, but rather transfer its cart (or other objects) to another partner. I see two cases:

1. Anonymous partner authenticates as an already known customer: we transfer the cart to the known customer and drop the anonymous partner.
2. Anonymous partner signs up: in that case create a new partner with sign-up information, and proceed as in case 1.

Reply:
You are as fast as V16 "blink of an eye" to answer to my comment ;).

Transfer can be a solution indeed (so we have the same flow as login with an existing one). Maybe we can "drop" the anonymous partner after the transfer.

Reply:
we implemented transfert here https://github.com/shopinvader/odoo-shopinvader/pull/1251

For the deletion of anonymous partner it can be done with a clean up cron or something.

Reply:
> anonymous partner should be inactive by default, we only active them when we fill real information.

why ?
A user can set a lang, an adress, etc... it's real info btw.

Reply:
It makes sense to me to have anonymous partners inactive to avoid polluting the contacts list.
I suspect users will always reach the anonymous partner forms by navigating from the carts.